code/core/src/main/java/com/adobe/marketing/mobile/services/ui/Presentable.java

```diff
@@ -0,0 +1,15 @@
+/*
+  Copyright 2023 Adobe. All rights reserved.
+  This file is licensed to you under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License. You may obtain a copy
+  of the License at http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software distributed under
+  the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+  OF ANY KIND, either express or implied. See the License for the specific language
+  governing permissions and limitations under the License.
+*/
+
+package com.adobe.marketing.mobile.services.ui;
+
+/** Represents a UI element which is both showable and dismissible */
+public interface Presentable extends Showable, Dismissible {}
```

Comment:
Are there any components that would implement show but not dismiss (or the other way around). More specifically, do we anticipate components that can only be shown and never dismissed? Asking because I am unclear on why `Showable` and `Dismissible` cannot be eliminated in favor of just having the `show()` and `dismiss()` in `Presentable` directly

Reply:
I am open to making that change. In this pr I was updating android to match the ios interfaces created.